docs/about.md

```diff
@@ -1,8 +1,8 @@
 # About Jellyfin
 
-The Jellyfin project was officially started on December 8th, 2018, in response to Emby's decision to take their 3.6 release closed-source, thus ending their remaining Free Software guise. The Jellyfin project history goes back further, to the first reports of GPL violations in Emby in late 2017, as well as the Nag screen issues in early 2017, as well as the over two-year trend of taking core pieces of Emby, including mobile apps and plugins, proprietary and refusing to release source.
+Several of the core members had been considering a fork since the initial inclusion of nag screens in 2016, along with the repeated closing of app sources and GPL license violations. The Jellyfin project was officially started on December 8th 2018 in response to Emby's decision to take their 3.6 release closed-source when the community finally got together around a common message. That we do care about the project and Free Software if we are welcomed and enabled to contribute!
 
-Jellyfin seeks to continue development of the original MediaBrowser project with a Free Software ethos. It is committed to bringing all its users access to the best possible Media System, developed entirely by the community.
+Jellyfin seeks to continue development of the original Emby project with a Free Software ethos. It is committed to bringing all its users access to the best possible Media System, developed entirely by the community.
```

Comment:
I chose MediaBrowser specifically since it was Emby's parent, but that's a fine change. Let's add idea the last line from above to the end:

>We welcome anyone who is interested in contributing!